wfw/tree.py

```diff
@@ -13,13 +13,35 @@
 
 
 class Tree(object):
-    def __init__(self, raw):
-        self.root = {'id' : "None", 'text' : 'My list', 'children' : [], 'done' : False}
-        children_of_root = raw['projectTreeData']['mainProjectTreeInfo']['rootProjectChildren']
+    # shared = False
+    # tree_data = None
+    # root = None
+
+    def __init__(self, tree_data, shared):
+        self._shared = shared
+        self._tree_data = tree_data
+        root = self.find_data_root()['rootProject']
```

Comment:
In case of my tree root is `None`, that is why I had hardcoded values for the root.

```{u'dateJoinedTimestampInSeconds': 00000000,
 u'initialMostRecentOperationTransactionId': u'00000000',
 u'initialPollingIntervalInMs': 10000,
 u'isReadOnly': False,
 u'itemsCreatedInCurrentMonth': 0,
 u'monthlyItemQuota': 0,
 u'ownerId': 000000,
 u'rootProject': None,
 u'rootProjectChildren': [{u'ch': [{u'id': u'21dsfs537-cdsdf-da74-sdfd9-09cxcvbthaea3',
                                    u'lm': 000000000,
                                    u'nm': u'text'
                                   {u'id': u'apsdfisad-004b-4a6c-8129-sdfasdfasdf',
```